pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch.go

```diff
@@ -33,6 +33,14 @@ import (
 	"istio.io/istio/pilot/pkg/util/runtime"
 )
 
+const (
+	// VirtualOutboundListenerName is the name for traffic capture listener
+	VirtualOutboundListenerName = "virtualOutbound"
```

Comment:
These are already defined in `listener.go`. Please use them rather than redefining here

Reply:
Unfortunately is not possible right now, the `envoyfilter` package is imported in `v1alpha3` where `listener.go` is.

Reply:
However, I can remove them from `listener.go` and use it from `envoyfilter` package in `v1alpha3`.